libraries/libbase/source/org/pentaho/reporting/libraries/base/util/SystemInformation.java

```diff
@@ -89,9 +106,12 @@ private String formatLongPathList( String value ) {
     return value;
   }
 
+  public Map<String, String> getOtherPropertiesInst() {
```

Comment:
No! Sorry, its leftover trash from when we where developing unit tests. I'll remove it.